packages/ckeditor5-core/src/editor/editor.ts

```diff
@@ -384,22 +384,23 @@ export default abstract class Editor extends /* #__PURE__ */ ObservableMixin() {
 
 			if ( !licenseKey ) {
 				/**
-				 * The `licenseKey` is missing in the editor configuration. If you use premium features,
-				 * please provide your license key. If you do not have a key yet, please contact us at
-				 * [https://ckeditor.com/contact/](https://ckeditor.com/contact/) or order a trial at
-				 * [https://orders.ckeditor.com/trial/premium-features](https://orders.ckeditor.com/trial/premium-features).
+				 * The `licenseKey` property is missing in the editor configuration.
 				 *
-				 * If you do not use premium features, add the `'GPL'` license key instead.
+				 * 	* If you are using the editor in a commercial setup, please provide your license key.
+				 * 	* If you still need to acquire a key, please [contact us](https://ckeditor.com/contact/) or
+				 *	  [start a free trial](https://portal.ckeditor.com/checkout?plan=free).
```

Comment:
```suggestion
				 *	  [create a free account with a 14 day premium features trial](https://portal.ckeditor.com/checkout?plan=free).
```

Just noticed we are not mentioning free accounts, just free trials.

Reply:
Fixed https://github.com/ckeditor/ckeditor5/pull/17400/commits/6ac6f1e53cbc8ff64874d2be942c2f2ac6fa8571